Makefile

```diff
@@ -42,16 +43,49 @@ test:
 basic_test:
 	go test $(BASIC_TEST_PKGS)
 
-check:
-	go get github.com/golang/lint/golint
-
-	@echo "vet"
-	@ go tool vet . 2>&1 | $(GOCHECKER)
-	@ go tool vet --shadow . 2>&1 | $(GOCHECKER)
-	@echo "golint"
-	@ golint ./... 2>&1 | $(GOCHECKER)
-	@echo "gofmt"
-	@ gofmt -s -l . 2>&1 | $(GOCHECKER)
+# setup needs to be ran just once to install dependencies
+setup:
+	go get github.com/twitchtv/retool
```

Comment:
We can move it into `tool-install`. `go get` will know it is already installed and skip the download&build process.

Reply:
I was trying to differentiate between global tools and those managed by retool. Right now on CI `setup` is ran  but `tool-install` is not. Even though `tool-install` is idempotent, it takes up time.